modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc

```diff
@@ -64,7 +64,7 @@ Complete the following fields for the *Basic* connection type:
 |Access Key |Alphanumeric text string that uniquely identifies the user who owns the AWS account.
 |Secret Key |Key that plays the role of a password.
 |Try Default AWS Credentials Provider Chain |(Optional) Indicates whether to use temporary credentials.
-|Region Endpoint |RL entry point for a web service.
+|Region Endpoint |URL entry point for a web service.
```

Comment:
Please look at all entries for this table column and, if all are fragments like this one, remove the periods.